src/request/FeedOptions.ts

```diff
@@ -1,47 +1,47 @@
-import { IHeaders } from "../index";
+import { CosmosHeaders } from "../index";
 
 /**
  * The feed options and query methods.
  */
-export interface FeedOptions {
+export class FeedOptions {
```

Comment:
Why make this a class?

Reply:
I think this is a mistake. I'm not sure why I did this.